### PR TITLE
Release v0.3.1

### PR DESCRIPTION
**Pull request by angrybayblade**
## Release summary

Version number: v0.3.1

## Release details

# Autonomy
- Factors out scaffolding templates
- Updates CLI help messages
- Fixes the scaffold `FSM` command so that a skill can be loaded once created
- Adds `valory/open-autonomy-user` Docker image with latest open-autonomy framework installed

# Packages
- Point to ACN Docker image instead of staging ACN nodes
- Removes unnecessary dependency of `transaction_settlement_abci` skill on the `offchain_aggregator` contract
- Adds environment variables for the RPC endpoints of all service specifications
- De duplicates `open_aea` packages by pulling from registry
- Increases the validate timeout by an order of magnitude in `Polygon` service

# Tests
- Fixes build image test

# Docs
- Adds a doc section to explain package management
- Fixes broken links on documentation

# Chores
- Fixes `release-image` target on the `Makefile` to use `packages.json`
- Check hashes in `packages.json` instead of `hashes.csv`
- Removes changelog
- Updates command regex to reflect latest changes

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side), from `develop`
- [x] I've updated the dependencies versions to the latest, wherever is possible.
- [x] Lint and unit tests pass locally (please run tests also manually, not only with `tox`)
- [x] I built the documentation and updated it with the latest changes
- [x] I've added an item in `HISTORY.md` for this release
- [x] I bumped the version number in the `__init__.py` file.
- [ ] I published the latest version on TestPyPI and checked that the following command work:
       ```pip install project-name==<version-number> --index-url https://test.pypi.org/simple --force --no-cache-dir --no-deps```
- [ ] After merging the PR, I'll publish the build also on PyPI. Then, I'll make sure the following
      command will work:
      ```pip install project-name-template==<version_number> --force --no-cache-dir --no-deps```  
- [ ] After merging the PR, I'll tag the repo with `v${VERSION_NUMVER}` (e.g. `v0.1.2`)

### CODE REVIEW

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
Do a search for `0.3.0`, you're missing a couple

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+- Factors out scaffolding templates
+- Updates CLI help messages
+- Fixes the scaffold `FSM` command so that a skill can be loaded once created
+- Adds `valory/open-autonomy-user` Docker image with latest open-autonomy framework installed
+
+# Packages
+- Point to ACN Docker image instead of staging ACN nodes
+- Removes unnecessary dependency of `transaction_settlement_abci` skill on the `offchain_aggregator` contract
+- Adds environment variables for the RPC endpoints of all service specifications
+- De duplicates `open_aea` packages by pulling from registry
```
Comment (dvilelaf) [inline HISTORY.md:16]:
```suggestion
- Deduplicates `open_aea` packages by pulling from registry
```

Comment (angrybayblade) [reply HISTORY.md:16]: This caused the spell check to fail

File: packages/valory/skills/liquidity_rebalancing_abci/skill.yaml [hunk 2]
```diff
@@ -147,5 +147,5 @@ models:
     class_name: SharedState
 dependencies:
   open-aea-test-autonomy:
```
Comment (DavidMinarsch) [inline packages/valory/skills/liquidity_rebalancing_abci/skill.yaml:149]: I am pretty sure more than one package uses this.

Also you didn't run generators

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**